catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpTransformer.java

```diff
@@ -0,0 +1,604 @@
+        List<String> ratingsList =
+                pathValueTracker.getAllValues(toPath(MgmpConstants.NIIRS_RATING_PATH));
+        if (CollectionUtils.isNotEmpty(ratingsList)
+                && CollectionUtils.isNotEmpty(ratingSystemList)) {
+            for (int i = 0; i < ratingsList.size() && i < ratingSystemList.size(); i++) {
+                if (ratingSystemList.get(i)
+                        .equals(MgmpConstants.NIIRS)) {
+                    try {
+                        Double niirsRating = Double.parseDouble(ratingsList.get(i));
+                        metacard.setAttribute(Isr.NATIONAL_IMAGERY_INTERPRETABILITY_RATING_SCALE,
+                                niirsRating);
+                    } catch (NumberFormatException e) {
+                        LOGGER.debug("Unable to parse double when setting NIIRS value", e);
+                    } finally {
+                        break;
+                    }
+                }
+            }
+        }
+    }
+
+    private void addMetacardResourceSecurityReleasability(
+            final XstreamPathValueTracker pathValueTracker, MetacardImpl metacard) {
+        addMetacardSecurityReleasability(pathValueTracker,
+                metacard,
+                MgmpConstants.RESOURCE_SECURITY_RELEASABILITY_PATH,
+                Security.RESOURCE_RELEASABILITY,
+                Security.RESOURCE_DISSEMINATION);
+    }
+
+    private void addMetacardMetadataSecurityReleasability(
+            final XstreamPathValueTracker pathValueTracker, MetacardImpl metacard) {
+        addMetacardSecurityReleasability(pathValueTracker,
+                metacard,
+                MgmpConstants.METADATA_RELEASABILITY_PATH,
+                Security.METADATA_RELEASABILITY,
+                Security.METADATA_DISSEMINATION);
+    }
+
+    private void addMetacardSecurityReleasability(final XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard, String path, String releasabilityConstant,
+            String disseminationConstants) {
+        String releasability = pathValueTracker.getFirstValue(toPath(path));
+        if (StringUtils.isNotEmpty(releasability)) {
+            addMetacardReleasabilityAndDisseminationControls(releasability,
+                    metacard,
+                    releasabilityConstant,
+                    disseminationConstants);
+        }
+    }
+
+    private void addMetacardReleasabilityAndDisseminationControls(String caveatString,
+            MetacardImpl metacard, String releasability, String disseminationControls) {
+        if (caveatString.startsWith(MgmpConstants.RELEASABLE_TO)) {
+            String[] nations = StringUtils.substringAfter(caveatString,
+                    MgmpConstants.RELEASABLE_TO + " ")
+                    .split("/");
+            List<String> nationsList = Arrays.asList(nations);
+            metacard.setAttribute(releasability, (Serializable) nationsList);
```

Comment:
Back to my comment on `SecurityAttributes.java`, you've defined the releasability attributes as single-valued and are setting lists.